setup.py

```diff
@@ -8,8 +8,10 @@
 def all_folders():
     folder_names = [
         "transformations/" + f
-        for f in list(OperationRuns.get_all_folder_names())
-    ]
+        for f in list(
+            OperationRuns.get_all_folder_names("transformations", "light")
```

Comment:
Though will use 'light' transformations only, I think it will be good if we take this as param in all_folders function and pass it from recursive_requirements i.e line no. 49. Because in case if the user wants to change the behaviour from 'light' to 'all' or something else, then will just need to change the param in line 49 instead of traversing the code and then changing it in line 12.

Reply:
Yes @aadesh11, I had made that changes when I made this PR a draft. Looks like some other issue is happening when we install light transformations only. Will fix the all the issues and push the changes again :)